ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -103,6 +103,14 @@ QtObject {
         globalUtilsInst.copyToClipboard(text)
     }
 
+    function copyImageToClipboard(content) {
+        globalUtilsInst.copyImageToClipboard(content)
+    }
+
+    function downloadImage(content, path) {
+        globalUtilsInst.downloadImage(content, path)
+    }
+
```

Comment:
Given that you've added this also in the root rootStore, is it really necessary to add the same logic here as well?

Reply:
I checked, and I think I found a way to reuse the ChatStore in the case of the PinnedMessagePopup. I'll try

Reply:
Ok, I updated. Now only the Chat store has those functions. I fixed it by passing the store when opening the PinnedMessagePopup instead of when the Component is declared.